src/bin/pg_autoctl/monitor.c

```diff
@@ -1518,65 +1529,60 @@ printCurrentState(void *ctx, PGresult *result)
 		return;
 	}
 
-	for (int i = 0; i <= maxHostNameSize; i++)
+	for (int i = 0; i < maxNameSize; i++)
 	{
-		if (i < maxHostNameSize)
-		{
-			nameSeparatorHeader[i] = '-';
-		}
-		else
-		{
-			nameSeparatorHeader[i] = '\0';
-		}
```

Comment:
What was this case used for and why is it not needed anymore now?

Reply:
We do a `bzero` before the loop now.